cmd/juju/crossmodel/show.go

```diff
@@ -14,16 +14,30 @@ import (
 )
 
 const showCommandDoc = `
-Show extended information about an offered application.
-
-This command is aimed for a user who wants to see more detail about what’s offered behind a particular URL.
+This command is intended to enable users to learn more about the
+application offered from a particular URL. In addition to the URL of
+the offer, extra information is provided from the readme file of the
+charm being offered.
 
 Examples:
-   $ juju show-offer fred/prod.db2
-   $ juju show-offer anothercontroller:fred/prod.db2
+To show the extended information for the application 'prod' offered
+from the model 'default' on the same Juju controller:
+
+     juju show-offer default.prod
+
+The supplied URL can also include a username where offers require them. This will be given as part of the URL retrieved from the
+'juju find-offers' command. To show information for the application
+'prod' from the model 'default' as the user 'admin':
```

Comment:
I didn't think this was *as* the user admin, this was *from* the user admin.
It still shows you the offer as seen by your login. It just allows you to see offers from models that other people are managing.